.github/workflows/test.yml

```diff
@@ -64,6 +69,11 @@ jobs:
         env:
           PUPPETEER_VERSION: ${{ matrix.puppeteer-version }}
 
+      - name: Run tests with remote browser
```

Comment:
this could be run in parallel to the other test, but it might not be worth it. But maybe something to try if this becomes too slow.